DataFormats/JetReco/interface/PileupJetIdentifier.h

```diff
@@ -57,12 +58,26 @@ class StoredPileupJetIdentifier {
 	DECLARE_VARIABLE(ptD        ,float);
 
 	DECLARE_VARIABLE(nvtx   ,float);  
+	DECLARE_VARIABLE(rho   ,float);
+	DECLARE_VARIABLE(fRing0    ,float);
```

Comment:
For the variables fRing0-3, axisMajor, axisMinor, nTot, nCh, DRweighted, pull,
you can avoid extending the dataformat, but instead reuse old variablenames as was done here:
https://github.com/cms-sw/cmssw/pull/10398/files#diff-34b721ce73bd0269273004088c8a715fR116